spec/services/proofing/socure/id_plus/request_spec.rb

```diff
@@ -0,0 +1,241 @@
+require 'rails_helper'
+
+RSpec.describe Proofing::Socure::IdPlus::Request do
+  let(:config) do
+    Proofing::Socure::IdPlus::Config.new(
+      api_key:,
+      base_url:,
+      timeout:,
+    )
+  end
+  let(:api_key) { 'super-$ecret' }
+  let(:base_url) { 'https://example.org/' }
+  let(:timeout) { 5 }
+  let(:user) { build(:user) }
+  let(:input) do
+    Proofing::Socure::IdPlus::Input.new(
+      email: user.email,
+      **Idp::Constants::MOCK_IDV_APPLICANT_WITH_PHONE.slice(
+        *Proofing::Socure::IdPlus::Input.members,
+      ),
+    )
+  end
+
+  subject(:request) do
+    described_class.new(config:, input:)
+  end
+
+  describe '#body' do
+    it 'looks right' do
+      freeze_time do
+        expect(JSON.parse(request.body, symbolize_names: true)).to eql(
+          {
+            modules: [
+              'kyc',
+            ],
+            firstName: 'FAKEY',
+            surName: 'MCFAKERSON',
+            dob: '1938-10-06',
+            physicalAddress: '1 FAKE RD',
+            physicalAddress2: nil,
+            city: 'GREAT FALLS',
+            state: 'MT',
+            zip: '59010-1234',
+            country: 'US',
+            nationalId: Idp::Constants::MOCK_IDV_APPLICANT_WITH_PHONE[:ssn],
+            countryOfOrigin: 'US',
+
+            email: user.email,
+            mobileNumber: Idp::Constants::MOCK_IDV_APPLICANT_WITH_PHONE[:phone],
+
+            userConsent: true,
+
+            # XXX: This should be set to the time the user submitted agreement,
+            #      which we are not currently tracking. The "5.minutes.ago" is
+            #      because Socure will reject times "in the future", so we avoid
+            #      our clocks being out of sync with theirs.
+            consentTimestamp: 5.minutes.ago.iso8601,
+          },
+        )
+      end
+    end
+  end
+
+  describe '#headers' do
+    it 'look right' do
```

Comment:
I was notorious for test names like this, but it should probably have a business-y title like `it 'includes the expected values'` or 'includes a SocureApiKey' or such.